src/models/userDao.js

```diff
@@ -0,0 +1,155 @@
+const { AppDataSource } = require('./dataSource');
+
+const findById = async (id) => {
+  const [user] = await AppDataSource.query(
+    `
+    SELECT
+      *
+    FROM
+      users
+    WHERE
+      id = ?
+    `,
+    [id]
+  );
+
+  return user;
+};
+
+const findBySellerId = async (sellerId) => {
+  const [user] = await AppDataSource.query(
+    `
+      SELECT
+        *
+      FROM
+        users
+      WHERE
+        seller_id = ?
+    `,
+    [sellerId]
+  );
+
+  return user;
+};
+
+const findByEmail = async (email) => {
+  const [user] = await AppDataSource.query(
+    `
+    SELECT
+      *
+    FROM
+      users
+    WHERE
+      email = ?
+    `,
+    [email]
+  );
+
+  return user;
+};
+
+const findSeller = async (name) => {
+  const [user] = await AppDataSource.query(
+    `
+    SELECT
+      *
+    FROM
+      sellers
+    WHERE
+      name = ?
+    `,
+    [name]
+  );
+
+  return user;
+};
+
+const findByKakao = async (kakaoId) => {
+  const [user] = await AppDataSource.query(
+    `
+    SELECT
+      *
+    FROM
+      users
+    WHERE
+      kakao = ?
+    `,
+    [kakaoId]
+  );
+
+  return user;
+};
+
+const signUp = async (
```

Comment:
```suggestion
const createUser = async (
```

- signup은 우리 서비스의 '기능' 이름이기 때문에 controller나 service 까지에서는 사용하기에 괜찮지만 dao에서는 마찬가지로 entity를 어떻게 조작하는 CRUD 기준에서 함수명을 명명하시는 것이 좋습니다

Reply:
넵 수정했습니다!